composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=5.3.6",
-        "fabpot/php-cs-fixer": "~1.11",
+        "friendsofphp/php-cs-fixer": "^1.11",
         "symfony/event-dispatcher": "~2.3|~3.0",
```

Comment:
There is a bug in php-cs-fixer 1.12 with `implement` on multiline: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2053
We should block the version to 1.11.*
This would also allow us to use the installed php-cs-fixer in travis for pim-community-dev CI, rather than downloading it with CURL.